Mapsui.Nts/Providers/Wfs/Xml/XPathQueryManager.cs

```
@@ -192,8 +193,12 @@ public IXPathQueryManager Clone()
             _paramContext.AddParam(queryParameters);
         string? result = null;
         FindXPath(xPath);
-        if (_xIter?.MoveNext() ?? false)
-            result = _xIter?.Current?.Value;
+        Catch.Exceptions(() =>
```

Why is this catch necessary? Could add be avoided somehow but first checking the data? Is this xml invalid? Is this a threading problem and should we fix that. In general, if there is this weird problem that you discovered while debugging that a comment needs to explain why this empty catch was added.

---

During testing this threw exceptions, I couldn't really find a way to detect the reason. Generally the WFSProvider throws to many exceptions in normal operating mode. I have to investigate it if I can reduce them, because when the exceptions are thrown at the wrong moment. Than the WFS Layer sample fails.

---

Okay, it is important to understand why.